simulation/Dockerfile

```
@@ -16,6 +16,7 @@ RUN apt-get update && apt-get install -y --no-install-recommends \
 # Install the simulator
 RUN wget coppeliarobotics.com/files/CoppeliaSim_Edu_V4_0_0_Ubuntu16_04.tar.xz && \
     tar -kxvf CoppeliaSim_Edu_V4_0_0_Ubuntu16_04.tar.xz && \
-    rm CoppeliaSim_Edu_V4_0_0_Ubuntu16_04.tar.xz
-COPY libsimExtROSInterface.so CoppeliaSim_Edu_V4_0_0_Ubuntu16_04/libsimExtROSInterface.so
-COPY servertest1.ttt CoppeliaSim_Edu_V4_0_0_Ubuntu16_04/servertest1.ttt
+    rm CoppeliaSim_Edu_V4_0_0_Ubuntu16_04.tar.xz && \
+    mv CoppeliaSim_Edu_V4_0_0_Ubuntu16_04 coppelia
```

```suggestion
    mv CoppeliaSim_Edu_V4_0_0_Ubuntu16_04 coppelia && \
    cp $(find coppelia -name libsimExtROSInterface.so | head -1) coppelia/libsimExtROSInterface.so
```

or you could just use the actual path `coppelia/compiledRosPlugins/libsimExtROSInterface.so`